src/lib/transaction_snark/test/account_timing/account_timing.ml

```diff
@@ -2045,8 +2045,7 @@ let%test_module "account timing check" =
                 ledger_init_state ;
               Transaction_snark_tests.Util.check_zkapp_command_with_merges_exn
                 ~expected_failure:
-                  Transaction_status.Failure
-                  .Update_not_permitted_timing_existing_account ledger
+                  Transaction_status.Failure.Update_not_permitted_timing ledger
```

Comment:
Can we include the update tests involving all possible permissions for sanity? I think including the following should work
```
module Test_input : Transaction_snark_tests.Test_zkapp_update.Input_intf =
struct
  let test_description = "timing"

  let failure_expected =
    Mina_base.Transaction_status.Failure. Update_not_permitted_timing

  let zkapp_update : Account_update.Update.t =
   let timing =
               Zkapp_basic.Set_or_keep.Set
                 ( { initial_minimum_balance =
                       Currency.Balance.of_nanomina_int_exn min_balance
                   ; cliff_time = Mina_numbers.Global_slot.of_int 1000
                   ; cliff_amount =
                       Currency.Amount.of_nanomina_int_exn 100_000_000
                   ; vesting_period = Mina_numbers.Global_slot.of_int 10
                   ; vesting_increment =
                       Currency.Amount.of_nanomina_int_exn 100_000_000
                   }
                   : Account_update.Update.Timing_info.value )
             in
             { Account_update.Update.dummy with timing } 
end

let%test_module "Update account timing" =
  ( module struct
    include Transaction_snark_tests.Test_zkapp_update.Make (Test_input)
  end )
```